plugins/modules/packaging/language/gem.py

```diff
@@ -123,6 +128,7 @@
 import re
 
 from ansible.module_utils.basic import AnsibleModule
+from packaging import version
```

Comment:
Most - if not all - the modules go with:
```
from distutils.version import LooseVersion
```
If I googled it right, you are referring to:
https://pypi.org/project/packaging/

This would add a dependency on the module - we try to avoid them as much as possible. Keep in mind that this code will be executed in the remote target, not on the controller (where you run `ansible-playbook` or AWX/Tower), so it would add a requirement to be `pip`-installed before that module can be used.